src/site/content/en/blog/badging-api/index.md

```diff
@@ -158,9 +153,52 @@ Android only ever shows a dot instead of a numeric value.
 Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
-then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
-accordingly.
+then the "+" won't appear. No matter the actual number, just call
+`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+displaying it accordingly.
+
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
+on the install state. Just call the API when it exists. If it works,
+it works. If not, it simply doesn't.
+
+## Supporting both the first and second origin trial {: #backwards-compat }
+
+In Chrome 79, the first origin trial ended, and based on developer feedback
+received during the first origin trial, was replaced by a new origin trial
+that uses a slightly different API design. Unfortunately, it is not backwards
+compatible, and calls to the previous API will fail.
+
+If you have an existing implementation, and want to provide backwards
+compatibility for your users, you can wrap the calls in the following
+wrapper function. Once all of your users have migrated to Chrome 79+, you
+can remove this wrapper function from your code.
+
+```js
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge) {
+    navigator.setExperimentalAppBadge.apply(navigator, args);
```

Comment:
@fallaciousreasoning pointed out that you can actually do this much more cleanly without `apply`:

```js
  if (navigator.setExperimentalAppBadge) {
    navigator.setExperimentalAppBadge(...args);
  } else if (window.ExperimentalBadge) {
    window.ExperimentalBadge.set(...args);
  }
```

Reply:
Done!